### PR TITLE
Set throttle to re-sync with new knobs

### PR DESCRIPTION
**Pull request by lsrinivas-pure**
**What this PR does / why we need it**:
- Adding options to update run rime options using parameters

- Minor fixes for logging

**Which issue(s) this PR fixes** (optional)
Closes #PTX-13132

**Special notes for your reviewer**:
```
STEP: Set throttle to re-sync
2022-11-02 19:27:16 +0530:[INFO dashboardutil.go::(*Dashboard).Infof:422]  Setting run time options: max_pending_resync_writes=2,blocks_per_resync_range=16
2022-11-02 19:27:16 +0530:[DEBUG restutil.go::getBody:158]  Response: {"status":"OK"}
2022-11-02 19:27:17 +0530:[DEBUG ssh.go::(*SSH).doCmdUsingPod.func1:611]  Running command on pod debug-9prjf [[nsenter --mount=/hostproc/1/ns/mnt /bin/bash -c which pxctl]]
2022-11-02 19:27:21 +0530:[DEBUG ssh.go::(*SSH).doCmdUsingPod.func1:611]  Running command on pod debug-9prjf [[nsenter --mount=/hostproc/1/ns/mnt /bin/bash -c sudo /usr/local/bin/pxctl cluster options update --runtime-options max_pending_resync_writes=2,blocks_per_resync_range=16]]
2022-11-02 19:27:23 +0530:[DEBUG portworx.go::(*portworx).SetClusterRunTimeOpts:3712]  Successfully set rt_opts
```

```
[root@torpedo-long-fa-sample-test-71-6 ~]# pxctl cluster options list
Default RPC timeout (minutes)                           : 5
Auto decommission timeout (minutes)                     : 20
Replica move timeout (minutes)                          : 1440
Replica move timestamp records threshold                : 134217728
Internal Snapshot Interval (minutes)                    : 30
Re-add timeout (minutes)                                : 1440
Resync repl-add                                         : off
Domain policy                                           : eventual
Cloudsnap optimized Restores                            : off
Cloudsnap Catalog                                       : off
Cloudsnap Abort Timeout (minutes)                       : 10
Cloudsnap Error Retry Limit                             : 3
Cloudsnap minimum Periodic schedule Interval (minutes)  : 15
Cloudsnap maximum threads                               : 16
Cloudsnap network interface                             :
Cloudsnap Using Metadata Enabled                        : on
Cloudsnap Metadata Upload Bytes limit                   : 10240 MiB
Cloudsnap Metadata Upload Percent limit                 : 15 %
Cloudsnaps Cleanup Failed Delay                         : 0
Cloudsnap full backup frequency                         : 7
Disable Provisioning Labels                             :
License expiry check (days)                             : 7
License expiry check interval                           : 6h0m5s
PxHttpProxy                                             :
Temporary Kvdb loss support                             : on
Unique Blocks size of volumes check interval            : 12h0m0s
Maximum concurrent API invocations                      : 20
Number of sharedv4 threads                              : 128
Sharedv4 (NFS) mount timeout in seconds                 : 130
Runtime options                                         : selector: -, options: blocks_per_resync_range=16,max_pending_resync_writes=2
ScheduleSnapshots Detached Volume Action                : optimized
Cache flush                                             : disabled
Cache flush interval in seconds                         : 30
Lttng last command executed                             :
Lttng disk usage setting (GB)                           : 0
Lttng loglevel setting                                  :
SkinnySnap                                              : off
SkinnySnap replication factor                           : 1
Cloudsnap cluster network usage limit                   : Not configured
RelaxedReclaim                                          : off
RelaxedReclaim maximum pending                          : 256
Volume expiration minutes                               : 0s
Auto Fstrim                                             : off
Max Fstrim IO rate                                      : 32 MiB
Min Fstrim IO rate                                      : 1 MiB
Max replicas per Node                                   : 4294967295
Pool Cache Configuration                                : off
```

### CODE REVIEW

File: tests/common.go [hunk 1]
```diff
@@ -4711,6 +4717,31 @@ func TeardownForTestcase(contexts []*scheduler.Context, providers []string, Clou
...
+
+		for _, opt := range runtimeOpts {
+			if !strings.Contains(opt, "=") {
+				dash.VerifyFatal(false, true, fmt.Sprintf("Given run time option is not in expected format key=val, Actual : %v", opt))
+			}
+			optArr := strings.Split(opt, "=")
+			optionsMap[optArr[0]] = optArr[1]
+		}
+		currNode := node.GetWorkerNodes()[0]
+		err = Inst().V.SetClusterRunTimeOpts(currNode, optionsMap)
```
Comment (nikolaypopov) [inline tests/common.go:4737]: You could just `return Inst().V.SetClusterRunTimeOpts(currNode, optionsMap)`

File: tests/testTriggers.go [hunk 2]
```diff
@@ -471,6 +471,11 @@ func TriggerDeployNewApps(contexts *[]*scheduler.Context, recordChan *chan *Even
 	Inst().M.IncrementCounterMetric(TotalTriggerCount, event.Event.Type)
 	Inst().M.SetGaugeMetricWithNonDefaultLabels(FailedTestAlert, 0, event.Event.Type, "")
 
+	Step(fmt.Sprintf("Set throttle to re-sync"), func() {
+		err := updatePxRuntimeOpts()
+		UpdateOutcome(event, err)
```
Comment (nikolaypopov) [inline tests/testTriggers.go:476]: Could just do `UpdateOutcome(event, updatePxRuntimeOpts())`

**Review by nikolaypopov — APPROVED** [review]
Otherwise looks ok, run torpedo with this before merging

File: tests/common.go [hunk 3]
```diff
@@ -3916,6 +3921,7 @@ func ParseFlags() {
 	flag.IntVar(&testsetID, testSetIDFlag, 0, "testset id to post the results")
 	flag.StringVar(&testBranch, testBranchFlag, "master", "branch of the product")
 	flag.StringVar(&testProduct, testProductFlag, "PxEnp", "Portworx product under test")
+	flag.StringVar(&pxRuntimeOpts, "px-runtime-opts", "", "run time options for cluster update")
```
Comment (jainsmit) [inline tests/common.go:3924]: It seems, this parameter support passing multiple runtime options, can we change the description with more information like `comma separated list of runtime options`

**Review by jainsmit — APPROVED** [review]
LGTM !! Except few comments

File: tests/common.go (hunk 3 above)
Comment (lsrinivas-pure) [reply tests/common.go:3924]: fixed

File: tests/common.go (hunk 1 above)
Comment (lsrinivas-pure) [reply tests/common.go:4737]: fixed

File: tests/testTriggers.go (hunk 2 above)
Comment (lsrinivas-pure) [reply tests/testTriggers.go:476]: fixed

**Pull request merged**